src/Workspaces/Core/Portable/TemporaryStorage/TemporaryStorageServiceFactory.cs

```diff
@@ -125,23 +125,21 @@ private MemoryMappedInfo CreateTemporaryStorage(long size)
         {
             // Larger blocks are allocated separately
             var mapName = CreateUniqueName(size);
-            var storage = MemoryMappedFile.CreateNew(mapName, size);
-            return new MemoryMappedInfo(new ReferenceCountedDisposable<MemoryMappedFile>(storage), mapName, offset: 0, size: size);
+            return new MemoryMappedInfo(mapName, offset: 0, size: size);
```

Comment:
>MemoryMappedInfo [](http://example.com/codeflow?start=23&length=16)

The old code called MMF.CreateNew, whereas the new code calls into MMF.OpenExisting. Is that OK?

Reply:
nope.  that was the bug :)